primitives/core/src/offchain/mod.rs

```diff
@@ -280,6 +280,8 @@ bitflags::bitflags! {
 		const TIME = 0b0001_0000_0000;
 		/// Access the statement store.
 		const STATEMENT_STORE = 0b0010_0000_0000;
+		/// Access to local node's mixnet key-exchange public keys.
+		const MIXNET_KX_PUBLIC_STORE = 0b0100_0000_0000;
```

Comment:
It is used in `execution_extensions.rs`. However I'm happy to switch away from the offchain worker as suggested, in which case this will not be needed.

Reply:
Done.